default.migration_vars.yml

```diff
@@ -1,18 +1,36 @@
-# Defaults used for migration playbook.
+# Personal or local values for migration playbook.
 # https://github.com/SU-SWS/drupal-caravan
-# ========================================
+# ================================================
 #
-
-# These should be in group_vars
-profile: "stanford"
-stack: "cardinal"
-stack_id: "2"
-server: "sites2"
-server_alias: "sse"
-site_prefix: "ds"
-
-# These can stay here
-localhost_home: ""
+# Save a copy of this file to the ansible-playbooks root directory
+# and name it, migration_vars.yml.
+#
+# Enter the environment sub-domain that you would like to run this
+# migration playbook against. `stanford` would be the production environment
+# `test-stanford` would be the test, or staging environment. And
+# `dev-stanford` would be the dev environment. The environment you choose
+# needs to match the environment choice below.
+sitefactory_environment: "test-stanford"
+#
+# Enter the enviroment drush alias prefix that you would like to run this
+# migration playbook against. `acsf` would be production. `acsf.test` would
+# be the test, or staging environment. And `acsf.dev` would be the dev
+# environment. The environment you choose needs to match the environment
+# choice above.
+drush_alias_environment: "acsf.test"
```

Comment:
It seems brittle to me that we could screw this up by entering conflicting values for `sitefactory_environment` and `drush_alias_environment`. Let's have Future Us make this more elegant.

Reply:
Let's have present us see if we can come up with a solution!  It felt like I was headed into the area of over-engineering a combined variable because the alias requires a period, that is not there in production, and the url requires a dash, that is not there in production.  But sometimes with these things two heads are better than one.

Reply:
I'm in favor of Now Us solving this, but maybe not in this PR. There's a lot of good work in here that I don't want to see blocked.

I created https://stanfordits.atlassian.net/browse/SITES-139 for followup.